.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+# https://www.appveyor.com/docs/appveyor-yml
+
+environment:
+  matrix:
+    - nodejs_version: 4.0
```

Comment:
Node is backwards compatible. This means Node v0.12 code works in Node v4, v9, etc. Therefore, since the lowest supported version is v4, this is the version that is tested, insuring the others will work as well.

Reply:
> Node is backwards compatible. This means Node v0.12 code works in Node v4, v9, etc.

The backwards compatibility on node mostly concerns on V8 and Node.js API backwards compatibility. However, every SEMVER-MAJOR of node might introduces breaking changes due to third_party dependencies.

There is an obscure example of breakage on the daily-used `\s` pattern:
```js
/\s/.test("\u180e")
// returns true on node.js 4.0
// returns false on node.js recent versions
```
The breakage is due to Unicode 6.3 [recategorize](http://unicode.org/L2/L2013/13004-vowel-sep-change.pdf) U+180E as format control. While node.js 4.0 depends on Unicode earlier than 6.3, later version will depends on later version since the latest ECMAScript always [use latest Unicode standard](https://github.com/tc39/ecma262/pull/620).

Not to detour, given the routine of the plugin is pretty simple and straightforward (we don't depends on Unicode Character Property in our regex), it is acceptable to test against node.js 4 only.

A practical node.js version principle should be testing against the minimum `postcss` supporting node.js version.

Another question is would `nodejs_version: 4.0` be resolved as `4.0.x` only on appveyor/travis? We should test against the latest nodejs 4.